app/views/schools/advice/electricity_recent_changes/_analysis.html.erb

```diff
@@ -0,0 +1,49 @@
+<p><%= t('advice_pages.electricity_recent_changes.analysis.summary') %></p>
+
+<p><%= t('advice_pages.electricity_recent_changes.analysis.sections') %></p>
+
+<ul>
+  <li><%= link_to(t('advice_pages.electricity_recent_changes.analysis.compare_recent_weeks.title'), '#compare-recent-weeks') %></li>
+  <li><%= link_to(t('advice_pages.electricity_recent_changes.analysis.compare_recent_days.title'), '#compare-recent-days') %></li>
+</ul>
+
+<!--  COMPARISON OF ELECTRICITY USE OVER 2 RECENT WEEKS -->
+<%= render 'schools/advice/section_title', section_id: 'compare-recent-weeks', section_title: t('advice_pages.electricity_recent_changes.analysis.compare_recent_weeks.title') %>
+
+<div class="charts">
+  <%= render(ChartComponent.new(
+    chart_type: @chart_config[:weekly],
+    school: @school,
+    analysis_controls: false,
+    no_zoom: true,
+    chart_config: @chart_config,
+    html_class: 'usage-chart')) do |c| %>
+    <% c.with_title { t('advice_pages.electricity_recent_changes.analysis.compare_recent_weeks.chart_title') } %>
+    <% c.with_subtitle { t('advice_pages.electricity_recent_changes.analysis.compare_recent_weeks.chart_subtitle') } %>
+    <% c.with_header do %>
+      <p><%= t('advice_pages.electricity_recent_changes.analysis.compare_recent_weeks.chart_description') %></p>
+    <% end %>
+  <% end %>
+  <%= render 'controls', chart_config: @chart_config, period: :weekly, supply: :electricity, split_meters: @split_meters, meters: @meters %>
```

Comment:
Would this work as the `footer` for the chart component?

Reply:
Aha yes I'd already made that change but hadn't pushed it..